### ISSUE TITLE
Atomic downgrade: Email notice after plan expiration and revert is complete

### ISSUE THREAD

**Issue by mmtr** [Automattic/wp-calypso#56059]
As noted in pdgTfM-3Y-p2, we need to send an email one week after plan expiration, informing Atomic sites that we have reverted their site and they lost access to plan features.

**Comment by onuro** [comment]
@mmtr we can fine tune the copy but let me know how that looks so far:

![Plan Expired](https://user-images.githubusercontent.com/645713/134345975-80909501-614e-44f8-ac3d-30fc7285bc57.png)

**Comment by mmtr** [comment]
Thanks @onuro, it looks great! As noted in pdgTfM-4r-p2, we're finally using the existing template to expedite a little bit our velocity. Once we ship a first iteration, we'll look into creating a new template according to your design.

**Comment by kdevnel** [comment]
I updated the title to better reflect the timing and intention of the email as per pdgTfM-4r-p2#comment-172

**Comment by dsas** [comment]
The requirement here is to send an email once we have deactivated an atomic enabling bundle plan (i.e. business/ecommerce), reverted the site to simple infra and restored any contents.

Out of scope:
 * Sending email when we have deactivated simple plans.
 * Sending email when deactivating, restoring and reverting due to deactivation of an atomic enabling marketplace product

This means sending the email will have to work asynchronously from the revert initiation. 

The way that current product lifecycle emails work/can be created is documented here: pdgTfM-3f-p2 Loading/filtering the people to send emails to relies upon the receipt item, receipt and ownership tables in SQL space and then further filtering records in PHP space. A disadvantage here is that the email isn't sent once we've finished changing their site, rather it would be sent the next day. We'd probably have to load all business/ecommerce subs that expired e.g. 8 days ago, and then in PHP space check that they are on simple still, then get their transfer record and look for some meta to know that they're eligible for an email need 

Another possible approach is to take a similar approach to D67909-code : 
 * In `deactivate_plan` before woa_revert, if the plan is one we want to send email for, get the transfer record and set a transfer meta to mark it as needing an email
 * Create a handler for `woa_revert_import_completed` that checks for the transfer meta mark and sends an email if it's there

The problem with that is that it's in a separate place to the rest of the lifecycle-related emails, and hooking it into the previewer etc might be a problem.
